fdbserver/ConsistencyScan.actor.cpp

```diff
@@ -858,7 +857,7 @@ ACTOR Future<Void> checkDataConsistency(Database cx,
 									firstKeySampledBytes += sampleInfo.sampledSize;
 
 								sampledKeys++;
-								if (itemProbability < 1) {
+								if (sampleInfo.probability > 0 && sampleInfo.probability < 1) {
```

Comment:
It shouldn't be possible that `inSample` is true if the probability is 0, right? And I guess even if it were 0, it shouldn't be a problem for anything in this block.

Reply:
This code is just incrementing a counter, so it shouldn't need any special handling of a 0 value